build.sbt

```diff
@@ -0,0 +1,82 @@
+val CatsEffectVersion = "3.4.8"
+val Log4CatsVersion = "2.5.0"
+val PureConfigVersion = "0.17.2"
+val Jms4SVersion = "0.0.1-53518bb-SNAPSHOT"
+
+ThisBuild / versionScheme := Some("semver-spec")
+ThisBuild / scalacOptions ++= Seq("-unchecked", "-deprecation")
+ThisBuild / githubTokenSource.withRank(KeyRanks.Invisible) := TokenSource.Or(
```

Comment:
This works for me in the CLI, but not with IntelliJ.  I have to add my own:

```
    resolvers += Resolver.githubPackages("rwalpole"),
    githubTokenSource := TokenSource.Or(
      TokenSource.Environment("GITHUB_TOKEN"),
      TokenSource.GitConfig("github.token") //  ~/.gitconfig
    )
```

Reply:
I have now modified this so it is the same as in [ctd-omega-editorial-frontend](https://github.com/nationalarchives/ctd-omega-editorial-frontend). Hopefully it works for you now?

Reply:
Confirmed.